css/style.scss

```diff
@@ -169,6 +169,7 @@ iframe {
       border-bottom: 5px solid #000;
       padding: 10px;
       margin: 0 0 10px;
+      clear: both;

```

Comment:
This will only kick in for the `(min-width: 768px)` breakpoint - is that what you're after?